docs/code/MlNetCookBook.md

```diff
@@ -1022,3 +1023,46 @@ ITransformer loadedModel;
 using (var fs = File.OpenRead(modelPath))
     loadedModel = newContext.Model.Load(fs);
 ```
+
+## How can I read and write binary data?
+Other than using text files ML.NET will allow you to read and write binary data. 
```

Comment:
>read and write binary data.  [](start = 53, length = 28)

Maybe a sentence on why we would do that: Smaller files, faster reading time, no need to specify the schema when you read it, etc.